Classes/DDFileLogger.m

```diff
@@ -869,6 +874,32 @@ - (void)maybeRollLogFileDueToSize {
     }
 }
 
+- (void)maybeRollLogFileDueToDate {
+	if (_rollEveryDay && _currentLogFileInfo && ![self isLogFileCreatedToday:_currentLogFileInfo]) {
+		NSLogVerbose(@"DDFileLogger: Rolling log file due to date...");
+		
+		[self rollLogFileNow];
+	}
+}
+
+- (BOOL)isLogFileCreatedToday:(DDLogFileInfo *)aLogFile {
+	NSDate *logFileCreationDate = aLogFile.creationDate;
+	NSDate *today = [NSDate date];
```

Comment:
This depends on the time zone of the user. So doesn't have much more meaning than rolling every 24 hours as in the current implementation.